apps/src/aichat/types.ts

```diff
@@ -9,6 +9,7 @@ export type ChatCompletionMessage = {
   chatMessageText: string;
   status: AichatInteractionStatus;
   timestamp?: string;
+  sessionId?: number;
```

Comment:
nit: might adding a comment on what this refers to (and why it's optional)?

Reply:
Yep good call, will do!